dev/kyuubi-extension-spark-common/src/main/scala/org/apache/kyuubi/sql/watchdog/ForcedMaxOutputRowsBase.scala

```diff
@@ -69,6 +69,7 @@ trait ForcedMaxOutputRowsBase extends Rule[LogicalPlan] {
       } else {
         true
       }
+    case _: MultiInstanceRelation => true
```

Comment:
why change this ?

Reply:
Place the ForcedMaxOutputRowsRule in the projectOptimizerRule stage，SQL like this: 
```
select * from table_a
```
this logicalPlan will be optimized as a `Relation` instead of a `Project`

Reply:
I see the project here is been removed since the output is same

Reply:
so we need to cover this situation with `case _: MultiInstanceRelation => true` ?

Reply:
I think it's fine if all test pass